ci/truffle_workaround.sh

```diff
@@ -1,16 +0,0 @@
-#!/bin/bash
```

Comment:
Interesting -- was this just a random find? Or is it related to the rest of the PR?

(Either way, good removal!!)

Reply:
This was a random find where one of the Hardhat fixtures failed and made me aware of this script